llvm/test/Analysis/LoopAccessAnalysis/allow-partial.ll

```diff
@@ -0,0 +1,99 @@
+; RUN: opt -disable-output -passes='print<access-info><allow-partial>,print<access-info>' %s 2>&1 | FileCheck %s --check-prefixes=ALLOW-BEFORE
```

Comment:
I don't think we have other print passes that take arguments already, but would it be more consistent to pare this as `print<access-info<allow-partial>>` to indicate this is an argument to access-info instead of `print<>`?